### PR TITLE
Added links to simple Golang wiki, pastebin and url shortener apps

### PR DESCRIPTION
**Pull request by prologic**
This PR will add the above software to the list.

### CODE REVIEW

**Comment by danger-awesome-Selfhosted** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 3 link issues</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
2 | Error | https://paste.mills.io <br> SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed
3 | Error | https://url.mills.io <br> SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed
4 | Error | https://wiki.mills.io <br> SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by prologic** [comment]
Is @ danger-awesome-Selfhosted really going to block this because of self-signed certs for Demo links?

File: README.md [hunk 1]
```diff
@@ -889,6 +889,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
   * [mojopaste](http://search.cpan.org/dist/App-mojopaste/) - Perl based pastebin. ([Demo](http://p.thorsen.pm/), [Source Code](https://github.com/jhthorsen/app-mojopaste)) `ARTv2` `Perl`
   * [NoteHub](https://notehub.org) - Free and Hassle-free Pastebin for Markdown Pages. Simple, clean, password provided, generated-short link. ([Demo](https://notehub.org), [Source Code](https://github.com/chmllr/NoteHub)) `MIT` `Nodejs`
   * [Paste](http://phpaste.sourceforge.io/) - Paste is forked from the original source pastebin.com used before it was bought. ([Demo](http://phpaste.sourceforge.net/demo), [Source Code](https://github.com/jordansamuel/PASTE)) `GPLv3` `PHP`
+  * [pastebin](https://github.com/prologic/pastebin) - pastebin is a simple pastebin service with convenient cli. ([Demo](https://paste.mills.io)) `MIT` `Go`
```
Comment (nodiscc) [inline README.md]: CLI in all caps as it is an acronym

File: README.md [hunk 2]
```diff
@@ -1144,6 +1145,7 @@ See https://staticsitegenerators.net and https://www.staticgen.com
   * [Lstu](https://github.com/ldidry/lstu) - _Let's SHorten That Url_ - Lightweight URL shortener. `WTFPL` `Perl`
   * [Nimbus](https://github.com/ethanal/nimbus) - A URL shortener and file sharer with a drag-and-drop OS X menu bar client and web interface. `MIT` `Python`
   * [Polr](https://project.polr.me/) - A modern, minimalist, modular, and lightweight URL shortener. ([Source Code](https://github.com/Cydrobolt/polr)) `GPLv2` `PHP`
+  * [shorturl](https://github.com/prologic/shorturl) - shorturl is a simple url shortener with very tiny urls. ([Demo](https://url.mills.io)) `MIT` `Go`
```
Comment (nodiscc) [inline README.md]: URL/URLs in all caps since they are acronyms

**Comment by nodiscc** [comment]
> Is @danger-awesome-Selfhosted really going to block this because of self-signed certs for Demo links?

It's not blocking, but yes, issuing a warning for self-signed certificate is fair. There is no valid trust chain so reporting `certificate verify failed` is correct.

**Comment by prologic** [comment]
Fair enough :) Not something I'm going to change anytime soon though. Waiting for [LeysEncrypt](https://letsencrypt.org/) to support wildcard certs!

Is this PR good to go?

**Comment by nodiscc** [comment]
> Is this PR good to go?

See the review above

**Comment by prologic** [comment]
Addressed feedback

**Comment by prologic** [comment]
Friendly ping? :)

**Comment by prologic** [comment]
bump

File: README.md [hunk 3]
```diff
@@ -1107,6 +1108,7 @@ See https://staticsitegenerators.net and https://www.staticgen.com
   * [Tasks](https://github.com/thewhitetulip/Tasks) - Kanban based to-do list manager written in Go. `MIT` `Go`
   * [tasks.php](https://github.com/RaymiiOrg/tasks.php) - A simple task/todo list manager. `MIT` `PHP`
   * [Taskwarrior](https://taskwarrior.org/) - Taskwarrior is Free and Open Source Software that manages your TODO list from your command line. It is flexible, fast, efficient, and unobtrusive. It does its job then gets out of your way. ([Source Code](https://taskwarrior.org/download/#git)) `MIT` `C++`
+  * [todo](https://github.com/prologic/todo) - todo is a simple todo manager ([Demo](https://todo.mills.io)) `MIT` `Go`
```
Comment (nodiscc) [inline README.md]: Missing final `.` after description

**Comment by prologic** [comment]
Fixed


James Mills / prologic

E: prologic@shortcircuit.net.au
W: prologic.shortcircuit.net.au

**Comment by prologic** [comment]
Sweet :)

File: README.md [hunk 4]
```diff
@@ -1107,6 +1109,7 @@ See https://staticsitegenerators.net and https://www.staticgen.com
   * [Tasks](https://github.com/thewhitetulip/Tasks) - Kanban based to-do list manager written in Go. `MIT` `Go`
   * [tasks.php](https://github.com/RaymiiOrg/tasks.php) - A simple task/todo list manager. `MIT` `PHP`
   * [Taskwarrior](https://taskwarrior.org/) - Taskwarrior is Free and Open Source Software that manages your TODO list from your command line. It is flexible, fast, efficient, and unobtrusive. It does its job then gets out of your way. ([Source Code](https://taskwarrior.org/download/#git)) `MIT` `C++`
+  * [todo](https://github.com/prologic/todo) - todo is a simple todo manager. ([Demo](https://todo.mills.io)) `MIT` `Go`
```
Comment (Kickball) [inline README.md]: This should be moved below TinyIssue.

**Comment by prologic** [comment]
Fixed :D


James Mills / prologic

E: prologic@shortcircuit.net.au
W: prologic.shortcircuit.net.au

**Comment by prologic** [comment]
Squashed commits

**Comment by Kickball** [comment]
@prologic, @danger-awesome-Selfhosted has highlighted a number of SSL errors on the demo sites, that I am also encountering.

Can these be resolved?

**Comment by prologic** [comment]
@Kickball Nope not easily :/ Not until [LetsEncrypt](https://letsencrypt.org/) start offering wild-card SSL certs which are apparently [coming](https://letsencrypt.org/2017/07/06/wildcard-certificates-coming-jan-2018.html)

**Comment by prologic** [comment]
@Kickball I take it back; I fixed the SSL issues wit some acme/letsencrupt magic! :)

**Pull request merged**